tests/test_broker.py

```diff
@@ -3,6 +3,9 @@
 
 from regolith.broker import Broker, load_db
 
+import tempfile
```

Comment:
this import should be above with `os` and subprocess.

Reply:
done